main.cpp

```diff
@@ -408,9 +408,11 @@ IntType& IntType::divide( int rhs )
     if (rhs == 0)
     {
         std::cout << "error: integer division by zero is an error and will crash the program!" << std::endl;
-        return *this;
     }
+    else
+    {
     *value /= rhs;
+    }
```

Comment:
Does this indentation look correct to you?
proofread your code before submitting to make sure it is as readable as possible
- [ ] revise